bluesky/plan_stubs.py

```diff
@@ -743,6 +748,12 @@ def inner_trigger_and_read():
             if hasattr(obj, 'trigger'):
                 no_wait = False
                 yield from trigger(obj, group=grp)
+        yield from create(streaming_name)
+        for obj in devices:
+            reading = (yield from read_streaming(obj))
```

Comment:
I guess a `read_streaming` plan stub needs to be defined also, before tests are added.

Reply:
oh yes, you're right... forgot to yield a `Msg`...thanks!